package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yargs",
-  "version": "17.6.2",
+  "version": "17.6.3-0",
```

Comment:
The version # is managed by automation, if you don't mind rolling this basck.

Reply:
running tests now, should have the PR updated within an hr or two at the latest

Reply:
All tests pass - as mentioned, happy to scrap the PR and do a fresh one for cleanliness, but either way - its good to go and rebased to the tip to include the other completion fix from yesterday